docs/src/content/docs/id/getting-started.mdx

```diff
@@ -73,57 +119,56 @@ description: This is a page in my Starlight-powered site
 ---
 ```
 
-Jika ada properti yang terlewat, Starlight akan memberitahu Anda.
-
-## Deploy website Starlight Anda
+Jika Anda lupa sesuatu yang penting, Starlight akan memberi tahu Anda.
 
-Setelah Anda selesai membuat dan mengkostumisasi website Starlight anda, anda dapat men-_deploy_ website Anda ke web server atau platform hosting pilihan Anda, termasuk Netlify, Vercel, GitHub Pages, dan lain sebagainya.
+### Langkah selanjutnya
 
-[Pelajari cara men-_deploy_ website Astro di dokumentasi Astro.](https://docs.astro.build/en/guides/deploy/)
+- **Mengkonfigurasi:** Pelajari tentang opsi umum dalam ["Menyesuaikan Starlight”](/id/guides/customization/).
+- **Navigasi:** Siapkan sidebar Anda dengan panduan ["Navigasi Sidebar”](/id/guides/sidebar/).
+- **Komponen:** Temukan kartu, tab dan bawaan lainnya di dalam panduan ["Komponen”](/id/guides/components/).
+- **Deploy:** Publikasikan karya Anda dengan panduan [“Deploy your site”](https://docs.astro.build/en/guides/deploy/) di dokumentasi Astro.
```

Comment:
Can "**Deploy:**" at the beginning be translated too or is this something that is better left in English?

Reply:
I think "Deploy" was left in English because it would be weird to translate, and "your site" I should have changed.